src/services/review/RepoCheckerService.ts

```diff
@@ -819,6 +852,29 @@ export default class RepoCheckerService {
     })
   }
 
+  permalinkNormaliser(permalink: string) {
+    let finalPermalink = permalink
+
+    if (finalPermalink.startsWith(`'`)) {
+      finalPermalink = finalPermalink.slice(1)
+    }
+    if (finalPermalink.endsWith(`'`)) {
+      finalPermalink = finalPermalink.slice(0, -1)
+    }
+    if (finalPermalink.startsWith(`"`)) {
+      finalPermalink = finalPermalink.slice(1)
+    }
+    if (finalPermalink.endsWith(`"`)) {
+      finalPermalink = finalPermalink.slice(0, -1)
+    }
+
+    if (!finalPermalink.startsWith("/")) {
+      finalPermalink = `/${finalPermalink}`
+    }
```

Comment:
This code looks weird 🤔 It looks like you expect to see values wrapped in both double quotes and backticks. Is that  normal?

Reply:
changed to regex pattern [6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85)

Reply:
commit changed after rebase [8bb11a1](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/8bb11a12e16b22fac09cfe5aaf1b4e79f919d0b6)